src/agent/src/cdh.rs

```diff
@@ -95,14 +82,48 @@ impl CDHClient {
 
 /// Init the cdh client
 pub async fn init_cdh_client() -> Result<()> {
-    let mut cdh_client = CDH_CLIENT.write().await; // Acquire write lock
+    let mut cdh_client = CDH_CLIENT.write().await;
```

Comment:
Either remove the comment on the previous commit, or leave the comment on this one. :-)

Reply:
Fixed.